test/compilable/ddoc15475.d

```diff
@@ -1,6 +1,10 @@
-// PERMUTE_ARGS:
-// REQUIRED_ARGS: -D -Dd${RESULTS_DIR}/compilable -o-
-// POST_SCRIPT: compilable/extra-files/ddocAny-postscript.sh
+/* PERMUTE_ARGS:
+REQUIRED_ARGS: -D -Dd${RESULTS_DIR}/compilable -o-
+POST_SCRIPT: compilable/extra-files/ddocAny-postscript.sh
+TEST_OUTPUT:
+---
+---
```

Comment:
Maybe this should be the default, and test should always fail a `TEST_NO_OUTPUT` check unless this is given.